scripts/toggle-packages.js

```diff
@@ -26,7 +27,8 @@ const getAllPackageJSON = async () => {
       packageList.push(
         ...dirs.map(dir => `${currentWorkspace}${dir}/package.json`)
       );
-    } else {
+    } else if (workspace.includes('examples/') && includeExamples) {
```

Comment:
Toggle only packages for development purposes. Examples should point at local builds for now, and will not need to change to prod builds